orbit/pkg/packaging/windows_templates.go

```diff
@@ -47,6 +47,7 @@ var windowsWixTemplate = template.Must(template.New("").Option("missingkey=error
                 <File Source="root\bin\orbit\windows\{{ .OrbitChannel }}\orbit.exe">
                   <PermissionEx Sddl="O:SYG:SYD:P(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;BU)" />
                 </File>
+                <Environment Id='OrbitUpdateInterval' Name='ORBIT_UPDATE_INTERVAL' Value='{{ .OrbitUpdateInterval }}' Action='set' System='yes' />
```

Comment:
Adding it as an environment variable (same as Linux and macOS) so that the change does not break using new `fleetctl` with an "old orbit" version. If you use arguments then you get the "unknown argument" error and Orbit won't start.